### PR TITLE
Add a SECURITY.md file

### PR DESCRIPTION
**Pull request by todb-r7**
Git has this cool shield button on the ribbon on every project now that indicates the vulnerability reporting policy for that project. We should totally populate this so people don't accidentally dox our bugs on Issues.

## Verification

It's docs, so just read it.

- [ ] Once landed, the link on the shield button up top should point to this doc.

### CODE REVIEW

**Comment by todb-r7** [comment]
I hope it's clear that I'm not married to any of this language at all. Mostly I just want to steer people away from GH Issues and into either ZenDesk, email, or heck even Keybase messaging or something about vulns in MSF and friends.

For me, it'd be super to write it in such a way that all Rapid7 projects can just use it as boilerplate, along with LICENSE.md and CONTRIBUTING.md (just let me have my fantasy that those are all replicated around all our projects, no need to correct me).

**Review by smcintyre-r7 — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+over email.
+
+## Via GitHub Issues
+
+Actually, please don't! Disclosing security vulnerabilities to public
+bug trackers is kind of mean, even when it's well-intentioned, since
+you end up dropping 0-day on pretty much everyone right out of the gate.
+We'd prefer you didn't!
+
+[r7vulns]:https://www.rapid7.com/security/disclosure/
```
Comment (smcintyre-r7) [inline SECURITY.md:33]: This link is actually broken, needs to be:
```suggestion
[r7-vulns]:https://www.rapid7.com/security/disclosure/
```

**Review by smcintyre-r7 — COMMENTED** [review]
I'd kind of prefer to see this under the [`.github`](https://docs.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository) folder to keep the root cleaner but it's not a strong preference.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+regardless.
+
+## Via email
+
+If you're more of a traditionalist, you can email your finding to
+security@rapid7.com. If you like, you can use our [PGP key][pgp] to
+encrypt your messages, but we certainly don't mind cleartext reports
+over email.
+
+## Via GitHub Issues
```
Comment (ccondon-r7) [inline SECURITY.md:26]: @todb-r7 Can we change the title to make it clearer (especially to non-English speakers!) that this _shouldn't_ be a disclosure path? e.g., maybe ## DON'T: Disclose Via GitHub Issues

**Comment by ccondon-r7** [comment]
Thanks for this, @todb-r7!

**Review by ccondon-r7 — APPROVED** [review]

**Comment by todb-r7** [comment]
> I'd kind of prefer to see this under the [`.github`](https://docs.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository) folder to keep the root cleaner but it's not a strong preference.

That's probably a better solution -- that would work for CONTRIBUTING.md and maybe a couple others, too, right? Maybe take a moment after this to sweep 'em all in there.... oh wow look at all this stuff you can [stash in .github](https://github.com/joelparkerhenderson/github_special_files_and_paths).

**Comment by smcintyre-r7** [comment]
# Release Notes
This adds a `SECURITY.md` file to the Metasploit Framework so users that have security issues know how they can be reported to the project maintainers. This file will be rendered through the GitHub interface as it follows the standard naming convention.

**Pull request merged**